fuzz/fuzz_targets/roundtrip.rs

```diff
@@ -40,12 +38,22 @@ enum AllTypes {
 }
 
 fuzz_target!(|data: &[u8]| {
-    let config = bincode::config::standard().with_limit::<1024>();
+    let config = bincode::config::legacy().with_limit::<1024>();
+    let configv1 = bincodev1::DefaultOptions::new().with_limit(1024).allow_trailing_bytes();
```

Comment:
This will use varint encoding, the `legacy()` will use fixint. The true legacy config from bincode 1 is `bincodev1::Config`